pkg/services/ngalert/provisioning/alert_rules.go

```diff
@@ -176,7 +176,11 @@ func (service *AlertRuleService) UpdateAlertRule(ctx context.Context, rule model
 	}
 	rule.Updated = time.Now()
 	rule.ID = storedRule.ID
-	rule.IntervalSeconds, err = service.ruleStore.GetRuleGroupInterval(ctx, rule.OrgID, rule.NamespaceUID, rule.RuleGroup)
+	// we need to use the stored rule namespace uid as the folder could have
+	// been renamed, thus the new namespace uid not matching the old one and
+	// no record would be found
+	rule.IntervalSeconds, err = service.ruleStore.GetRuleGroupInterval(ctx, rule.OrgID,
```

Comment:
Possibly dumb question, but why do we need to look up the interval from the DB if we already have `storedRule`? Can we not just use `storedRule.IntervalSeconds` straight up?

Reply:
Good catch, changed it.